docs/tutorials/getting-started-with-flask.rst

```diff
@@ -19,10 +19,6 @@ Finally, create a new directory for this tutorial and go inside it:
    mkdir flask-hello-world
    cd flask-hello-world
 
-Note that you'll also need a text editor. You can either install one of your
-choice or simply use one of the already existing editors in your Ubuntu
-environment (like ``vi``).
```

Comment:
Already included in the literalinclude above

Reply:
true